client/src/index.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {BrowserRouter as Router, Route, Link} from 'react-router-dom';
```

Comment:
wherever possible, only import what you need as it creates extra work for the compilar and slows down performance